tests/test_inmemorychannel.py

```diff
@@ -62,7 +62,6 @@ async def test_multi_send_receive(channel_layer):
     """
     Tests overlapping sends and receives, and ordering.
     """
-    channel_layer = InMemoryChannelLayer()
```

Comment:
The recreation of `channel_layer` looks intentional to me, what do you mean when you say the tests become bogus?

Reply:
nope, channel_layer is provided by a fixture. It makes absolutely no sense to create a channel_layer again.

(the fixture creates already fresh channel_layers)